### PR TITLE
Update 5490 Benefit Eligibility "Important" Text

### PR DESCRIPTION
**Pull request by cehsu**
UPDATED:
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5458#issuecomment-343318500

Spouse Important text:
screen shot 2017-11-16 at 10 15 24 pm

Child Important text:
screen shot 2017-11-16 at 10 15 44 pm

### CODE REVIEW

File: src/js/edu-benefits/5490/helpers.jsx [hunk 1]
```diff
@@ -50,14 +50,8 @@ export const benefitsRelinquishedWarning = (
...
 );
 
-export const benefitsDisclaimerSpouse = (
+export const benefitsDisclaimer = (
   <p>
-    IMPORTANT: If you are eligible for Chapter 35 Survivors’ and Dependents’ Educational Assistance Program (DEA) and eligible for Chapter 33 Post-9/11 GI Bill Marine Gunnery Sergeant John David Fry Scholarship (Fry Scholarship), you must relinquish entitlement to the benefit that you are not applying for (even if entitlement arises from separate events). <strong>You cannot retain eligibility for both programs simultaneously</strong>.
-  </p>
-);
-
-export const benefitsDisclaimerChild = (
```
Comment (cehsu) [inline src/js/edu-benefits/5490/helpers.jsx]: @peggygannon These were previously two different messages, is there now only one?

File: src/sass/modules/_m-schemaform.scss [hunk 2]
```diff
@@ -17,6 +17,10 @@ legend {
   }
 }
 
+input[name="root_benefit"] { 
+  width: 4rem;
+}
```
Comment (cehsu) [inline src/sass/modules/_m-schemaform.scss]: This was necessary to enable the cursor to more conveniently access the AdditionalInfo button.

File: src/js/edu-benefits/5490/config/form.js [hunk 3]
```diff
@@ -125,13 +124,13 @@ const formConfig = {
...
             'view:benefitsDisclaimerChild': {
-              'ui:description': benefitsDisclaimerChild,
+              'ui:description': benefitsDisclaimer,
               'ui:options': {
                 hideIf: form => _.get('relationship', form) !== 'child'
               }
             },
             'view:benefitsDisclaimerSpouse': {
-              'ui:description': benefitsDisclaimerSpouse,
+              'ui:description': benefitsDisclaimer,
```
Comment (cehsu) [inline src/js/edu-benefits/5490/config/form.js]: Depending on @peggygannon's response below, view:benefitsDisclaimerChild and view:benefitsDisclaimerSpouse may need to be consolidated.

**Comment by jbalboni** [comment]
The additional info pattern on those pages looks a little cluttered to me. Curious what @stacyhlinn thinks.

**Comment by stacyhlinn** [comment]
@jbalboni I see what you mean. When this pattern was designed, we didn't really take into account having so many additional info patterns on the same page. I wonder if spacing out the bullet point items a tad more would help? Would that make it inconsistent with the bullet point pattern though?

**Comment by cehsu** [comment]
Thanks @jbalboni @stacyhlinn, in order to remain consistent with the existing radio pattern but provide more space between tooltips, would something like this be an option?
screen shot 2017-11-10 at 10 28 02 am

File: src/js/edu-benefits/5490/helpers.jsx (hunk 1 above)
Comment (peggygannon) [reply src/js/edu-benefits/5490/helpers.jsx]: @cehsu Hi, Claire, the message I am editing is the one at the top of the page: apply-for-education-benefits/application/5490/benefits/eligibility. IMPORTANT: If you are eligible for Chapter 35 Survivors’ and Dependents’ Educational Assistance Program (DEA) and eligible for Chapter 33 Post-9/11 GI Bill Marine Gunnery Sergeant John David Fry Scholarship (Fry Scholarship), you must relinquish entitlement to the benefit that you are not applying for (even if entitlement arises from separate events). You cannot retain eligibility for both programs simultaneously.

I only see one message there. I did not edit the conditional messages that pop up when you choose either DEA or Fry.

Comment (peggygannon) [reply src/js/edu-benefits/5490/helpers.jsx]: @cehsu Hi, Claire, the message I'm editing is the one at the top of the page: 
IMPORTANT: If you are eligible for Chapter 35 Survivors’ and Dependents’ Educational Assistance Program (DEA) and eligible for Chapter 33 Post-9/11 GI Bill Marine Gunnery Sergeant John David Fry Scholarship (Fry Scholarship), you must relinquish entitlement to the benefit that you are not applying for (even if entitlement arises from separate events). You cannot retain eligibility for both programs simultaneously.

I didn't edit the conditional messages that pop up if the user chooses DEA or Fry.

**Comment by stacyhlinn** [comment]
@cehsu: I had a chat with @peggygannon and we both agree that for the page that asks "Which benefit are you currently using?", we should delete the "Find out if you're eligible for xxxxx" links at the end of each paragraph. It doesn't seem relevant when we're asking what benefits they're already using. 

As far as the design, i will update you once I get the design team to sign off on the new additional info pattern.

**Comment by cehsu** [comment]
Thanks @stacyhlinn, I've removed the "Find out if you're eligible for xxxxx" links and updated the unfurled screenshot above.

**Comment by cehsu** [comment]
Thanks for your feedback @peggygannon, kindly refer to the updated screenshots provided above.

**Review by jbalboni — APPROVED** [review]

**Review by peggygannon — COMMENTED** [review]
@ceshu. LGTM

**Pull request merged**